gcc/rust/checks/errors/rust-feature-gate.cc

```diff
@@ -131,7 +131,8 @@ FeatureGate::visit (AST::MacroRulesDefinition &rules_def)
 void
 FeatureGate::visit (AST::Function &function)
 {
-  check_rustc_attri (function.get_outer_attrs ());
+  if (!function.is_external ())
+    check_rustc_attri (function.get_outer_attrs ());
```

Comment:
Why disable this check?

Reply:
I believe this check is for Function only and not External Functions,  without  `if (!function.is_external ())` compiler was not able detect `#![feature(intrinsics)]` in case of external function attributes as in `issue-1031.rs`.